vue.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  lintOnSave: false
```

Comment:
We don't want to auto-lint on file save?

Should we set up a pre-commit hook like husky to make sure changes are linted before commit?